lib/defaults.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
         babelOptions: {
             babelrc: false,
             parserOpts: {
-                plugins: ['classProperties', ['decorators', { decoratorsBeforeExport: false }]],
```

Comment:
Sorry, seems like I added this but it's been a while. Was this unnecessary to begin with, or maybe it's no longer required because class fields are now added by default?

Reply:
> Was this unnecessary to begin with, or maybe it's no longer required because class fields are now added by default?

The latter, now that fields reached stage 4 the flag is not needed anymore: https://github.com/babel/babel/commit/ceaab0bae717ec96c43d9e875b9d2de25e57007c#